test/nuts/trackingCommands/mpd-sequential.nut.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/* eslint-disable no-console */
+
+import * as path from 'path';
+import { fs, AuthInfo, Connection } from '@salesforce/core';
+import { expect } from 'chai';
+import { TestSession, execCmd } from '@salesforce/cli-plugins-testkit';
+import { replaceRenamedCommands } from '@salesforce/source-tracking';
+import { PushResponse } from '../../../src/formatters/pushResultFormatter';
+
+let session: TestSession;
+let conn: Connection;
+
+describe('end-to-end-test for tracking with an org (single packageDir)', () => {
```

Comment:
this is using an MPD project, not `(single packageDir)` right?

do we need a NUT testing `"pushPackageDirectoriesSequentially": false`? Are we validating that `"true"` and `true` result in the same behavior?

Reply:
1. it's *the* mpd project :)
2. having a non-sequential mpd is a good idea.
3. "true" would result in it not being mpd.  It's really a boolean, not a boolean | string.  Maybe that's worth a discussion...